runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -303,11 +341,11 @@ public void onGameTick(final GameTick event)
 
 		if (clue instanceof NpcClueScroll)
 		{
-			String npc = ((NpcClueScroll) clue).getNpc();
+			List<String> npcs = ((NpcClueScroll) clue).getNpcs();
 
-			if (npc != null)
+			if (!npcs.isEmpty())
 			{
-				Query query = new NPCQuery().nameEquals(npc);
+				Query query = new NPCQuery().nameEquals(npcs.toArray(new String[0]));
```

Comment:
If you make getNpcs and getObjectIds and all that jazz into arrays, you can just pass in npcs here.